common/web/es-bundling/src/tslibTreeshaking.mts

```diff
@@ -42,7 +42,7 @@ export async function determineNeededDowncompileHelpers(config: esbuild.BuildOpt
 
   let tslibHelperDetectionPlugin = {
```

Comment:
Possibly should rename this now that 'tslib' has disappeared (es-bundling is a better name so that's good)